src/api.js

```diff
@@ -0,0 +1,19 @@
+const API_END_POINT = 'http://0.0.0.0:3000/';
+
+export const API = {
+  UPLOAD_RESUME: '/resume/upload',
+};
+
+export const request = async (uri, options = {}) => {
+  try {
+    const fullUri = `${API_END_POINT}${uri}`;
+    const response = await fetch(fullUri, options);
+    if (response.ok) {
+      const json = await response.json();
+      return json;
+    }
+    throw new Error('API통신 실패 => response.ok :', response.ok);
+  } catch (e) {
+    console.error('API통신 실패 :', e.message);
+  }
+};

```

Comment:
이 함수는 utills 에서 관리 되어야 겠네요
```suggestion
export const request = async (uri, options = {}) => {
  try {
    const response = await fetch(uri, options);
    if (response.ok) {
      const json = await response.json();
      return json;
    }
    throw new Error('API통신 실패 => response.ok :', response.ok);
  } catch (e) {
    console.error('API통신 실패 :', e.message);
  }
};
```
api 객체가 수정되서 fullUri은 받지 않고, 바로 uri로 넘겨줘도 되겠네요